### PR TITLE
Use Strftime to improve fixed timestamp performance

### PR DESCRIPTION
**Pull request by repeatedly**
Use `Strftime` instead of `Time#strftime` improves the performance of time formatting.

### CODE REVIEW

**Comment by repeatedly** [comment]
@mururu Could you also check this?

File: lib/fluent/plugin/formatter_stdout.rb [hunk 1]
```diff
@@ -21,13 +21,14 @@ module Plugin
...
 
-      TIME_FORMAT = '%Y-%m-%d %H:%M:%S.%9N %z'
+      TIME_FORMAT = '%Y-%m-%d %H:%M:%S.%N %z'
 
       config_param :output_type, :string, default: 'json'
 
       def configure(conf)
         super
 
+        @time_formatter = Strftime.new(TIME_FORMAT)
```
Comment (mururu) [inline lib/fluent/plugin/formatter_stdout.rb]: Don't we need `rescue nil` here?

File: lib/fluent/log.rb [hunk 2]
```diff
@@ -163,21 +163,22 @@ def logdev=(logdev)
...
           r << msg
           r
         }
       when :json
         @format = :json
-        @time_format = '%Y-%m-%d %H:%M:%S %z'
         @formatter = Proc.new { |type, time, level, msg|
           r = {
-            'time' => time.strftime(@time_format),
+            'time' => @time_formatter ? @time_formatter.exec(time) : time.strftime(@time_format),
```
Comment (mururu) [inline lib/fluent/log.rb]: How about defining method for `@time_formatter ? @time_formatter.exec(time) : time.strftime(@time_format)`?

Comment (repeatedly) [reply lib/fluent/log.rb]: Done.

File: lib/fluent/plugin/formatter_stdout.rb (hunk 1 above)
Comment (repeatedly) [reply lib/fluent/plugin/formatter_stdout.rb]: Yes because TIME_FORMAT is fixed.

Comment (mururu) [reply lib/fluent/plugin/formatter_stdout.rb]: I see.

**Comment by mururu** [comment]
Looks good.

**Pull request merged**